src/views/order_form_view.js

```diff
@@ -0,0 +1,67 @@
+import Backbone from 'backbone';
+import _ from 'underscore';
+import Order from '../models/order';
+import OrderView from '../views/order_view';
+import Quote from '../models/quote';
+import QuoteView from '../views/quote_view';
+
+const OrderFormView = Backbone.View.extend({
+  initialize(params){
+    this.template = params.template;
+    this.bus = params.bus;
+    this.quotesList = params.quotesList;
```

Comment:
I like that the `OrderFormView` knows about the `QuoteList` so it can associate the quote with a new order.